### PR TITLE
Add release step before deploy

### PR DESCRIPTION
**Pull request by catalin-apostu**
## Issue reference

N/A

## Small description of change

Add release step before deploy.

## Type of Change

- [ ] Fix
- [x] Feature
- [ ] Documentation Updates
- [ ] Breaking Changes

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -31,7 +30,7 @@ jobs:
         if: "contains(github.event.head_commit.message, '[minor]')"
         run: echo "RELEASE=minor" >> $GITHUB_ENV
 
-      - run: yarn
+      - run: yarn build
```
Comment (endriu3314) [inline .github/workflows/release.yml:33]: Why does it need `build`?
We run `yarn` to install the modules (basically just `release-it`) then just run `release-it` to bump and release on GitHub.

**Review by endriu3314 — CHANGES_REQUESTED** [review]
Maybe keep name of `release.yml` to `release_and_bump.yml` or something as it also does that

File: .github/workflows/release.yml (hunk 1 above)
Comment (catalin-apostu) [reply .github/workflows/release.yml:33]: It should not release if it cannot build the application. For example, if multiple PRs being merged lead to broken code.

**Review by endriu3314 — APPROVED** [review]
LGTM

**Pull request merged**